src/test/scala/InstanceRestartingSpec.scala

```diff
@@ -0,0 +1,136 @@
+import com.tapad.docker.DockerComposeKeys._
+import com.tapad.docker._
+import org.mockito.Matchers._
+import org.mockito.Mockito._
+import org.scalatest.{ BeforeAndAfter, FunSuite, OneInstancePerTest }
+import scala.collection.Iterable
+
+class InstanceRestartingSpec extends FunSuite with BeforeAndAfter with OneInstancePerTest with MockHelpers {
```

Comment:
You may not need "BeforeAndAfter" since you didn't put any prereq or cleanup methods in the tests.